### ISSUE TITLE
Minimalistic UI

### ISSUE THREAD

**Issue by paales** [GraphCMS/Feature-Requests#34]
The current UI for adding content is very heavy.

<img width="1290" alt="schermafbeelding 2017-06-30 om 10 09 38" src="https://user-images.githubusercontent.com/1244416/27726840-4c39cefc-5d7c-11e7-9a45-eb984461ffc4.png">

The above essentially are three fields which can be reduced to simple form fields.

**Comment by mlukaszczyk** [comment]
Thanks again for the feedback. We will think about how to make it more compact. One of the next steps regarding the UI will be, that it won't take the full screen of a view. Instead it should slide out from the right so that the content table should still be visible on the left side. This will make content editing a lot faster on larger screens.

**Comment by eruby94** [comment]
So the issue of horizontal space has been solved for larger screens, as you mentioned @mlukaszczyk .

What remains unaddressed, however, is making the vertical space more compact. This is generally against material design, which seems to guide much of the interface. I do agree however, since any model with more than about 5 fields takes the full height of a laptop screen, and several models will have many more than that.

It should ideally be a toggle, where you can enable / disable 'compact mode' similar to some text editor UI's, where font size, margins, padding, and line-height are decreased, without forcing the user to zoom out using their browser.

**Comment by paales** [comment]
@eruby94 I don't think it is about a compact UI. It is about removing bloat, panels, dropshadows, etc. e.g.:

- Icon can be removed
- The 'tags' below the label can all be removed/simplied
- Dropshadow can be gone
- 'Length:' can be removed, numbers alone are enough
- Expand/collapse seems useless if it doesn't actually remember anything throughout sessions?

**Comment by eruby94** [comment]
Perhaps it is both. I agree with all you have mentioned. Regardless, a settings toggle would be nice to have for 'preference' of the individual user. Some may like to see the 'bloat' visual aesthetic

**Comment by mlukaszczyk** [comment]
Agree on both views here. We are currently working on some nice features and we are in the process of creating a new component library in parallel that will take this feedback into account.

**Comment by belazer** [comment]
This will be addressed in the new GraphCMS web app coming soon :)
